lib/command.c

```diff
@@ -2228,6 +2240,9 @@ DEFUN (config_log_syslog,
        LOG_LEVEL_DESC)
 {
 	int idx_log_levels = 2;
+
+	disable_log_file();
```

Comment:
NAK. syslog, stdout and file are independent and can be active at the same time.